site/src/pages/WorkspacePage/Workspace.stories.tsx

```diff
@@ -95,6 +95,51 @@ export const PendingInQueue: Story = {
 	},
 };
 
+export const PendingWithNoProvisioners: Story = {
+	args: {
+		...Running.args,
+		workspace: {
+			...Mocks.MockPendingWorkspace,
+			latest_build: {
+				...Mocks.MockPendingWorkspace.latest_build,
+				matched_provisioners: {
+					count: 0,
+					available: 0,
+				},
+			},
+		},
+	},
+};
+
+export const PendingWithNoAvailableProvisioners: Story = {
+	args: {
+		...Running.args,
+		workspace: {
+			...Mocks.MockPendingWorkspace,
+			latest_build: {
+				...Mocks.MockPendingWorkspace.latest_build,
+				matched_provisioners: {
+					count: 1,
+					available: 0,
+				},
+			},
+		},
+	},
+};
+
+export const PendingWithUndefinedProvisioners: Story = {
+	args: {
+		...Running.args,
+		workspace: {
+			...Mocks.MockPendingWorkspace,
+			latest_build: {
+				...Mocks.MockPendingWorkspace.latest_build,
+				matched_provisioners: undefined,
```

Comment:
I am curious: when is this a valid/expected state?

Reply:
https://github.com/coder/coder/blob/32aec0b07be5596403f56d81f7cad8114ff17db1/site/src/testHelpers/entities.ts#L1202-L1206

Reply:
The values would depend on what a user has configured, but this is one example of a healthy payload.